docker-hub/vulnerability-scanning.md

```diff
@@ -1,124 +1,174 @@
 
-> **Scan report retention period**
->
-> Docker retains the vulnerability scan report for one year. If you'd like a new, up-to-date vulnerability scan report after this period, you can rescan the image by pushing the image to Docker Hub.
+4. Push the image to Docker Hub to trigger Basic vulnerability scanning on the
+   image:
 
-To view the vulnerability report:
+   ```console
+   $ docker push <your-Docker-ID>/<your-repo-name>:latest
+   ```
 
-1. Go to [Docker Hub](https://hub.docker.com){: target="_blank" rel="noopener" class="_"} and open the repository to view a summary of the vulnerability scanning report.
+## View the vulnerability report
 
-    It can take a couple of minutes for the vulnerability report to appear in your repository.
+To view the vulnerability report:
 
-    ![Vulnerability scan report](images/vuln-scan-report.png){:width="700px"}
+1. Go to [Docker Hub](https://hub.docker.com){: target="_blank" rel="noopener"
+   class="_"} and open the repository page to view a summary of the Basic
+   vulnerability scanning report.
 
-2. Click on the **Tags** tab > **Digest** > **Vulnerabilities** to view the detailed scan report.
+   It may take a couple of minutes for the vulnerability report to appear in
+   your repository.
 
-    The scan report displays vulnerabilities identified by the scan, sorting them according to their severity, with highest severity listed at the top. It displays information about the package that contains the vulnerability, the version in which it was introduced, and whether the vulnerability is fixed in a later version.
+   ![Vulnerability scan report](images/vuln-scan-report.png){:width="700px"}
 
-    ![Vulnerability scan details](images/vuln-scan-details.png){:width="700px"}
+2. Click on the **Tags** tab > **Digest** > **Vulnerabilities** to view the
+   detailed scan report.
 
-    > **Note**
-    >
-    > You can also click the ‘open in browser’ icon  on each row to view detailed information about the vulnerability on the Snyk website.
+   The scan report displays vulnerabilities identified by the scan, sorting them
+   according to their severity, with highest severity listed at the top. It
+   displays information about the package that contains the vulnerability, the
+   version in which it was introduced, and whether the vulnerability is fixed in
+   a later version.
 
-### Disable vulnerability scanning
+   ![Vulnerability scan details](images/vuln-scan-details.png){:width="700px"}
 
-Repository owners and administrators can disable vulnerability scanning on a repository. To disable scanning:
+For more information on this view, see
+[Image details view](../scout/image-details-view.md).
 
-1. Log into your [Docker Hub](https://hub.docker.com){: target="_blank" rel="noopener" class="_"} account.
-2. Click **Repositories** from the main menu and select a repository from the list.
-3. Go to the **Settings** tab and click **Disable Image Scanning**.
+### Turn off Basic vulnerability scanning
```

Comment:
Super picky point, but I'd move this down to the bottom, so each heading is presented in the order a user would generally go in: enable VS, inspect, fix, turn off VS